app/src/main/java/com/willowtree/vocable/room/PhraseDao.kt

```diff
@@ -0,0 +1,22 @@
+package com.willowtree.vocable.room
+
+import androidx.room.*
+
+@Dao
+interface PhraseDao {
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    suspend fun insertPhrase(phrase: Phrase)
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    suspend fun insertPhrases(vararg phrases: Phrase)
+
+    @Delete
+    suspend fun deletePhrase(phrase: Phrase)
+
+    @Query("SELECT * FROM Phrase WHERE is_user_generated == 1")
```

Comment:
Not sure if this breaks anything but is it supposed to be `WHERE is_user_generated == 1` or `WHERE is_user_generated = 1`?

Reply:
Cuz this is based on mySQL comparisons and not kotlin/java lang right?

Reply:
It looks like Room accepts both `=` and `==` here